src/components/DifferentCards/CartProductBig/NormalCard.jsx

```diff
@@ -1,15 +1,14 @@
 import "./Cart-Product-Big.css";
 import { BsSuitHeartFill } from "react-icons/bs";
 import { TiTickOutline } from "react-icons/ti";
-
 import { Link, useNavigate, useLocation } from "react-router-dom";
 import { useDispatch, useSelector } from "react-redux";
 import {
   addToCart,
   deleteItemFromWishlist,
   postItemToWishlist,
-} from "./../../../Redux/Reducers-Redux/operationsSlice";
-// Small product Card
+} from "../../../Redux/Reducers-Redux/operationsSlice";
```

Comment:
can use absolute path instead of relative path

Reply:
Yeah done it thanks a lot for reviewing pr